hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/fair/FSQueue.java

```diff
@@ -97,10 +97,11 @@ public FSQueue(String name, FairScheduler scheduler, FSParentQueue parent) {
 
   /**
    * Initialize a queue by setting its queue-specific properties and its
-   * metrics. This function don't set the policy for queues since there is
-   * different logic to do that.
-   * This function is invoked when a new queue is created or reloading the
-   * allocation configuration.
+   * metrics. This method is invoked when a new queue is created or reloading
```

Comment:
Minor language comment. Let us use one of the following two:
- when creating a new queue or reloading the allocation file.
- when a new is created or the allocation file is reloaded.